### PR TITLE
feat: add support for interactive args

### PR DESCRIPTION
**Pull request by abhijeetSaroha**
I have implemented a feature to enable interactive arguments using the `input` method instead of `prompt`. The reason for choosing `input` is to avoid the square brackets ([]) displayed during prompts. For example:

* with `prompt`:
```bash
username []:
password []:
```

* with `input`:
```bash
username:
password:
```

I have tested the feature. Additionally, I have created a test file for this implementation. Please review and test from your side whenever you have time.

Solves #42

### CODE REVIEW

**Comment by abhijeetSaroha** [comment]
@xmnlab Please review it.

**Comment by xmnlab** [comment]
@abhijeetSaroha could you rebase your branch on top of the upstream/main please²

File: .makim.yaml [hunk 1]
```diff
@@ -73,16 +73,17 @@ groups:
...
-          - task: smoke-tests.complex
-          - task: smoke-tests.containers
-          - task: smoke-tests.unittest
-          - task: smoke-tests.vars-env
-          - task: smoke-tests.test-vars
-          - task: smoke-tests.bash
-          - task: smoke-tests.dir-absolute-path
-          - task: smoke-tests.dir-no-path
-          - task: smoke-tests.dir-relative-path
+          # - task: smoke-tests.simple
```
Comment (xmnlab) [inline .makim.yaml:76]: Any reason for disabling all thse tests?

Comment (abhijeetSaroha) [reply .makim.yaml]: No...I just testing the interactive-args...so I comment down other test cases and forget to un-comment them before committing the changes. Sorry..

Comment (abhijeetSaroha) [reply .makim.yaml]: I have commit the new changes.

**Comment by xmnlab** [comment]
@abhijeetSaroha thanks for working on that!
could you change it to use `prompt` instead of `input`. prompt is much more robust than input, and it will allow us to use types and default easily.

the issue you mentioned:

```
username []:
password []:
```

that happens because the default is an empty string. you can test with:

```
import click

click.prompt('Type your name', type=str, default="myname")
click.prompt('Type your name', type=str, default="")
click.prompt('Type your name', type=str)
```

so for now that is not an issue, maybe in the future we could just try to change the way it is displayed.

**Comment by abhijeetSaroha** [comment]
I made the required changes. It is working fine with `prompt`.

**Comment by xmnlab** [comment]
thanks for working on that @abhijeetSaroha

**Pull request merged**